components/raftstore-v2/src/raft/peer.rs

```diff
@@ -173,6 +175,63 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         self.region().get_id()
     }
 
+    /// Set the region of a peer.
+    ///
+    /// This will update the region of the peer, caller must ensure the region
+    /// has been preserved in a durable device.
+    pub fn set_region(
```

Comment:
Is this supposed to be put here? Seems only simple getter/setters are placed in this file.

Reply:
So, which file you think is better?

Reply:
I don't know really, just a thought.